package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apps-script-oauth1",
-  "version": "1.15.0",
+  "version": "1.16.0",
```

Comment:
Please revert the version change, I'll take care of that when I release a new version.

Reply:
Done.